src/views/index.js

```diff
@@ -820,7 +820,7 @@ exports.settingsView = ({ status, peers, theme, themeNames, version }) => {
     })
   );
 
-  const languageOption = (shortName, longName) =>
```

Comment:
how do you feel about `({longName, shortName})` to help reduce ordering issues?

Reply:
Yep, named parameters via object destructuring are my preference. I'd merge a patch to change that. :+1: